src/standalone/DataGrid/Content/Cell.tsx

```diff
@@ -82,19 +83,19 @@ const Cell = (props: CellProps): React.ReactElement => {
 			onMouseLeave={endHover}
 			onClick={toggleSelection}
 			onDoubleClick={editRecord}
-			className={`${classes.cell} ${
-				props.rowIndex !== 0 ? classes.dataCell : classes.headerCell
-			} ${
+			className={combineClassNames([
+				classes.cell,
+				props.rowIndex !== 0 ? classes.dataCell : classes.headerCell,
```

Comment:
There shouldn't be any empty strings passed to combineClassNames

The following code will evaluate to false instead of empty string in case rowIndex is 0
```tsx
props.rowIndex !== 0 && (props.rowIndex % 2 === 0 ? classes.rowEven : classes.rowOdd)
```

Reply:
@TQjona , updated changes. Please Review.